jac-uk/admin
# 1776/agency report barrister

## What's included?
this includes all candidates who have declared barrister experience in the BSB report, both on the admin site, and in the download.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
[Go to this exercise](https://jac-admin-develop--pr1915-1776-agency-report-b-pymsu4ql.web.app/exercise/11bv7ZTyyKhoqGYAPN7Y/reports/agency#bsb) under reports > agency report > BSB
- Under BSB ensure all candidates who have selected yes to barrister appear.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
